### PR TITLE
resolve pip vulnerability

### PR DESCRIPTION
**Pull request by quchuyuan**
Python (Pip) Security Update for requests (GHSA-9wx4-h78v-vm56) for the image public/azureml/curated/minimal-app-quickstart

### CODE REVIEW

File: assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ dependencies:
 - pip:
   # vulnerabilities
   - cryptography>=41.0.3
```
Comment (vizhur) [inline assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml:10]: remove cryptography to wheel, there should be no need having those in conda spec

**Review by vizhur — CHANGES_REQUESTED** [review]

File: assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ dependencies:
 - pip:
```
Comment (vizhur) [inline assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml:8]: update pip to something recent

File: assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml (hunk 1 above)
Comment (quchuyuan) [reply assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml]: removed

File: assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml (hunk 2 above)
Comment (quchuyuan) [reply assets/inference/environments/minimal-app-quickstart/context/conda_dependencies.yaml:8]: done

**Review by vizhur — APPROVED** [review]

**Pull request merged**